homeassistant/components/music_assistant/media_player.py

```diff
@@ -276,22 +276,36 @@ async def async_on_update(self) -> None:
             self._attr_state = MediaPlayerState(player.state.value)
         else:
             self._attr_state = MediaPlayerState(STATE_OFF)
+
         group_members_entity_ids: list[str] = []
+        coordinator_id: str | None = None
+        group_childs: set[str] = set()
         if player.group_childs:
+            # This player is the coordinator, with children players
+            coordinator_id = self.player_id
+            group_childs = set(player.group_childs) - {self.player_id}
```

Comment:
a set is unordered so please use a list instead of set if you lik eto keep the order

Reply:
Also note that group_childs can also be set for a group player so maybe adjust the comment

Reply:
As raised in a later comment, this was to re-order later since it was assumed that `group_childs` was an unordered set. I'll update this